backend/tests/v5/e2e/routes/users.test.js

```diff
@@ -433,6 +441,82 @@ const testDeleteApiKey = () => {
 	});
 };
 
+const testForgotPassword = () => {
+	describe('Send forgot password email', () => {
+		test('should fail if a username or email is not provided', async () => {
+			const res = await agent.post('/v5/user/password').expect(templates.invalidArguments.status);
+			expect(res.body.code).toEqual(templates.invalidArguments.code);
+		});
+
+		test('should return ok even if user does not exist', async () => {
+			await agent.post('/v5/user/password').send({ user: 'non existing user' })
+				.expect(templates.ok.status);
+		});
+
+		test('should send forgot password email with valid username', async () => {
+			await agent.post('/v5/user/password').send({ user: testUser.user })
+				.expect(templates.ok.status);
+		});
+	});
+};
+
+const testResetPassword = () => {
+	describe('Reset user password', () => {
+		test('should fail if a token is not provided', async () => {
+			const res = await agent.put('/v5/user/password').send({ newPassword: 'Abcdefghijk1!', user: testUserWithToken.user })
```

Comment:
Don't have to change this (unless you want to)

but should use `generateRandomString()` on any strings you don't care about. (that's what it's for afterall!)
```suggestion
			const res = await agent.put('/v5/user/password').send({ newPassword: generateRandomString(), user: testUserWithToken.user })
```